src/hooks/useBuyout/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { OrmlTraitsAssetRegistryAssetMetadata } from './types';
+
+const AMPE_HARDCODED_METADATA: Record<string, OrmlTraitsAssetRegistryAssetMetadata> = {
```

Comment:
We don't have USDT on Foucoco, so let's have a new separate `FOUCOCO_HARDCODED_METADATA` object which only has the same record for KSM but not the one for USDT.